src/classes/issues-processor.ts

```diff
@@ -911,6 +922,48 @@ export class IssuesProcessor {
     return this.options.removeStaleWhenUpdated;
   }
 
+  private async _removeLabelsWhenUpdatedFromStale(
+    issue: Issue,
+    labels: Readonly<string>[]
+  ): Promise<void> {
+    if (!labels.length) {
+      return;
+    }
+
+    const issueLogger: IssueLogger = new IssueLogger(issue);
+
+    issueLogger.info(
+      `Removing any labels specified via remove-labels-when-updated-from-stale...`
+    );
+
+    for (const label of labels.values()) {
+      await this._removeLabel(issue, label);
+    }
+  }
+
+  private async _addLabelsWhenUpdatedFromStale(
+    issue: Issue,
+    labels: Readonly<string>[]
+  ): Promise<void> {
+    if (!labels.length) {
+      return;
+    }
+
+    const issueLogger: IssueLogger = new IssueLogger(issue);
+
+    issueLogger.info(
+      `Adding labels marked as add-labels-when-updated-from-stale...`
+    );
+
+    this._statistics?.incrementAddedItemsLabel(issue);
```

Comment:
Not sure if this is the statistic to increment

Reply:
This is a statistic to reflect a call to GitHub to add (one or multiple - we don't care since it's an array) a label to an issue/PR, so it seems OK to me, but I will have a better insight once I reviewed it.